### ISSUE TITLE
Configure a node to relay another node's subnet

### ISSUE THREAD

**Issue by icy** [tailscale/tailscale#2977]
Hi all, 

During Pandemic time, my team has to use two hops as below

```
                          vv exit node
{home network}  -->  {jumphost1} ---> {jumphost2} --> {internal subnet}
                                           ^^ tailscale up --advertise-routes=internal-subnet
```

The idea behind is that the direct connection between `{home network}` and `{jumphost2}` is very unstable, while all other routes are very fast (`{home network}` to `{jumphost1}`, `{jumphost1}` to `{jumphost2}`, and `{jumphost2}` to `{internal subnet}`).

What I  can do now is to run `tailscale up --advertise-routes=internal-subnet` command and also configure `{jumphost1}` as an `exit node`.

My question is whether another option to tell client to access our internal subnet over the both jumphost without any exit nodes?

Thanks for your reading and support.

**Comment by bradfitz** [comment]
Tailscale's whole goal is to do away with all such historical jumpbox complexity and work end-to-end simply.

It's very unlikely we'll add such support, as it seems like it's solving the wrong problem. (I'm not even sure I buy the problem, either... if your first and only hop out of your house is bad, why do you expect things to be better if you move other parts around?)

**Comment by icy** [comment]
Thanks for your time reading my issue. Fyi, ` {jumphost1}` has some robust connection because the team can buy it from some hosting provider. At their home , the network is just not good to some places in the world; some user can't even have a chance to switch their ISP. We can setup some ipsec tunnels but that sounds a lot of work.

The right problem to solve is something like this https://vietnaminsider.vn/vietnams-submarine-cable-system-broken-down-once-again/ though.

**Comment by DentonGentry** [comment]
Tailscale is not likely to fix submarine cables either.


For your specific question: as far as I can see it isn't necessary to be an exit node, you could have jumphost1 --advertise-routes=internal-subnet just as a subnet router.

It would be up to you to configure the route tables to get the traffic to jumphost2 and thence to the internal subnet.

**Comment by icy** [comment]
> Tailscale is not likely to fix submarine cables either.

No I don't expect that. I was raising the question, because tailscale gives an option to select `exit node`, but at the same time, doesn't have an option to select `its subnet router`. It is just awesome if both "router" can be treated the same way from client.  But maybe I didn't ask correctly/well...

> For your specific question: as far as I can see it isn't necessary to be an exit node, you could have jumphost1 --advertise-routes=internal-subnet just as a subnet router.
>
> It would be up to you to configure the route tables to get the traffic to jumphost2 and thence to the internal subnet.

I think I already mentioned ipsec in my previous comment. Technically it's possible without tailscale. The traditional setup is actually the first thing we think of.

It sounds strange to see some submarine issues, but that's actually happening all the time. Many people and team have the same issues, and you know what they pick up  Openvpn,ipsec or alike most of the time. I actually really like the idea the idea of wireguard and tailscale (I spent quite amount of time to read our blog https://tailscale.com/blog/how-nat-traversal-works/ ) I am also pushing the team hard to try tailscale if possible, and avoid all the hassle setup.

Thanks again for your reading my issue and taking your team to reply.

**Comment by shshe** [comment]
Hi, I stumbled on this issue and we have a use case where this might be useful.

We use Tailscale to connect to our AWS VPC - our primary VPC has a public internet and NAT gateway. We're connecting to a second AWS account through an AWS VPC Endpoint (a.k.a. PrivateLink) - this second VPC has no public internet or NAT gateway. Ideally, I'd like to use Tailscale on our primary VPC to connect into machines within the subnets of the second VPC, but tunneled through the VPC Endpoint of the primary.

Let me know if there's a way to accomplish this today. Curious if there's an easier way to do this than what I described.
